src/Umbraco.Infrastructure/Composing/HostBuilderExtensions.cs

```diff
@@ -25,6 +26,7 @@ public static IHostBuilder UseUmbraco(this IHostBuilder builder)
         /// <param name="umbracoServiceProviderFactory"></param>
         /// <returns></returns>
         public static IHostBuilder UseUmbraco(this IHostBuilder builder, UmbracoServiceProviderFactory umbracoServiceProviderFactory)
-            => builder.UseServiceProviderFactory(umbracoServiceProviderFactory);
+            => builder.UseServiceProviderFactory(umbracoServiceProviderFactory)
+                .UseSerilog();
```

Comment:
what is this doing?

Reply:
Created this follow up task: https://dev.azure.com/umbraco/D-Team%20Tracker/_workitems/edit/7757